types/App.ts

```diff
@@ -0,0 +1,26 @@
+type App = {
+  id: number;
+  name: string;
+  description?: string;
+  portalId: number;
+  updatedAt: number;
+  createdAt: number;
+  clientId: string;
+  iconUrl?: string;
+  archived: boolean;
+  ownerId?: number;
+  isUserLevel: boolean;
+  isBusinessUnitEnabled: boolean;
+  isFeatured: boolean;
+  isInternal: boolean;
+  documentationUrl?: string;
+  supportUrl?: string;
+  supportEmail?: string;
+  supportPhone?: string;
+  extensionIconUrl?: string;
+  isAdvancedScopesSettingEnabled: boolean;
+};
+
+export type FetchAppsResponse = {
+  results: Array<App>;
+};
```

Comment:
I made a `types/app` file recently that this could be moved to. Also made a type for `PublicApp` - you may be able to use that or make some kind of generic `App` type and then extend it

Reply:
https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/types/Apps.ts

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/130/commits/fbd7671efc9a0dafef58bd063c610257dc976f99.

We were importing the same endpoint, so I deleted my work and opted for yours 👍